### ISSUE TITLE
LC0017 in preview doesn't respect trailing comments

### ISSUE THREAD

**Issue by tkroes** [StefanMaron/BusinessCentral.LinterCop#856]
I have a piece of code that writes to a FlowField on a temporary table, which is copied from the Base Application. These lines had trailing comments on the same line, but LC0017 was still triggering using the preview build. I would expect a trailing comment to also be valid for clearing up this diagnostic.

Not flagged:
```al
//Copied from standard for temporary applying entry
TempApplyingCustLedgEntry.Amount := GenJnlLine.Amount;
```

Flagged in current preview version:
```al
TempApplyingCustLedgEntry.Amount := GenJnlLine.Amount; //Copied from standard for temporary applying entry
```

**Comment by Arthurvdv** [comment]
Thank you for reporting this, it seems I've unknowingly broken this in the upcoming release 😳

I've resolve this and added extra test cases on this for the future.
